### DISCUSSION TITLE
When fully switching to lazer, what will happen when it will be fully released?

### DISCUSSION THREAD

**Discussion by akkiirah** [ppy/osu#17562] [Q&A] [answered]
I'm thinking about fully switching to lazer already.
But currently, beatmaps, skins etc cannot be found in in lazer's folders.
How will the update from stable to lazer impact players that have already switched complety to lazer?
Will all beatmaps, skins etc become visible in the explorer and it is possible to just keep using it or will it be needed to upgrade from stable to lazer importing everything again?

In case this is a duplicate I'm sorry, didn't find any info regarding this so far.

**Comment by peppy** [comment, marked as answer]
You will need to run the import process again. It will skip any already imported items.
